mix.exs

```diff
@@ -12,7 +12,7 @@ defmodule Appsignal.Mixfile do
   use Mix.Project
 
   @source_url "https://github.com/appsignal/appsignal-elixir"
-  @version "2.2.7"
+  @version "2.2.8"
```

Comment:
I don't quite understand from your comment about the version kerfuffle, but this version bump is done by mono, you don't need to update it in this PR, or at least not in the merged version.

Reply:
Right! In that case, is it okay for the pull requests on the Phoenix and Plug repositories to point to a not-yet-released version of `appsignal-elixir`?

Reply:
Yes :)